.ibm/pipelines/openshift-ci-tests.sh

```diff
@@ -152,47 +159,49 @@ run_tests() {
   (
     set -e
     echo Using PR container image: ${TAG_NAME}
-    yarn test
+    yarn "$project"
   ) |& tee "/tmp/${LOGFILE}"
 
   RESULT=${PIPESTATUS[0]}
 
   pkill Xvfb
 
-  mkdir ${ARTIFACT_DIR}/test-results
-  cp -a /tmp/backstage-showcase/e2e-tests/test-results/* ${ARTIFACT_DIR}/test-results
+  mkdir -p ${ARTIFACT_DIR}/$project/test-results
+  cp -a /tmp/backstage-showcase/e2e-tests/test-results/* ${ARTIFACT_DIR}/$project/test-results
 
   ansi2html <"/tmp/${LOGFILE}" >"/tmp/${LOGFILE}.html"
   cp -a "/tmp/${LOGFILE}.html" ${ARTIFACT_DIR}
   exit ${RESULT}
 }
 
 check_backstage_running() {
-  local url="https://${RELEASE_NAME}-backstage-${NAME_SPACE}.${K8S_CLUSTER_ROUTER_BASE}"
-
-  # Maximum number of attempts to check URL
-  local max_attempts=30
-  # Time in seconds to wait
-  local wait_seconds=30
-
-  echo "Checking if Backstage is up and running at $url" | tee "/tmp/${LOGFILE}"
-
-  for ((i=1; i<=max_attempts; i++)); do
-    # Get the status code
-    local http_status=$(curl --insecure -I -s "$url" | grep HTTP | awk '{print $2}')
-
-    # Check if the status code is 200
-    if [[ $http_status -eq 200 ]]; then
-      echo "Backstage is up and running!"
-      export BASE_URL=$url
-      echo "######## BASE URL ########"
-      echo "$BASE_URL"
-      return 0
-    else
-      echo "Attempt $i of $max_attempts: Backstage not yet available (HTTP Status: $http_status)" | tee -a "/tmp/${LOGFILE}"
-      sleep $wait_seconds
-    fi
-  done
+  local release_name=$1
+    local namespace=$1
```

Comment:
code in this check_backstage_running function does not appear to be properly formatted. there are extra white spaces.